scale/docs/rest/recipe.rst

```diff
@@ -409,3 +409,259 @@ These services provide access to information about recipes.
+| **Content Type**   | *application/json*                                                                                 |
++--------------------+----------------------------------------------------------------------------------------------------+
+| **JSON Fields**                                                                                                         |
++--------------------+-------------------+----------+---------------------------------------------------------------------+
+| job_names          | Array[String]     | Optional | The name of jobs within the recipe definition that should be        |
+|                    |                   |          | included in the reprocessing request, even when the definition for  |
+|                    |                   |          | those jobs has not changed between recipe type revisions.           |
++--------------------+-------------------+----------+---------------------------------------------------------------------+
+| all_jobs           | Boolean           | Optional | A flag that indicates all jobs in the recipe should be reprocessed, |
+|                    |                   |          | even when the recipe type definitions are identical. This option    |
+|                    |                   |          | overrides any job_name parameters and is typically used to re-run   |
+|                    |                   |          | jobs that failed due to temporary errors.                           |
++--------------------+-------------------+----------+---------------------------------------------------------------------+
+| .. code-block:: javascript                                                                                              |
+|                                                                                                                         |
+|    {                                                                                                                    |
+|        "all_jobs": true                                                                                                 |
+|    }                                                                                                                    |
++-------------------------------------------------------------------------------------------------------------------------+
+| **Successful Response**                                                                                                 |
++--------------------+----------------------------------------------------------------------------------------------------+
+| **Status**         | 201 CREATED                                                                                        |
++--------------------+----------------------------------------------------------------------------------------------------+
+| **Content Type**   | *application/json*                                                                                 |
```

Comment:
I think in other API docs, we listed the Location header and explained what the URL pointed to.